example/src/main/scala/example/AuthenticationClient.scala

```diff
@@ -0,0 +1,21 @@
+package example
+
+import zhttp.http.Headers
+import zhttp.service.{ChannelFactory, Client, EventLoopGroup}
+import zio.{App, ExitCode, URIO}
+
+object AuthenticationClient extends App {
```

Comment:
We should provide more documentation on this Client App. What this app is doing and how to use it.

Reply:
Done